### DISCUSSION TITLE
how to get "dump.ch4" and "bonds.reaxc.ch4_new" files?

### DISCUSSION THREAD

**Discussion by McAlexDudu** [tongzhugroup/mddatasetbuilder#140] [Q&A] [answered]
Hi, when I use datasetbuilder, and find the command "datasetbuilder -d dump.ch4 -b bonds.reaxc.ch4_new -a C H O -n ch4 -i 25" seem to need two files. Must I run lammps or give some tips? thanks.

**Comment by njzjz** [comment, marked as answer]
The dump file is necessary to provide coordinates. The bonds file is optional. The bond order can also be generate by coordinates.

————————
Jinzhe Zeng
PhD candidate in Chemistry
Graduate Assistant, Institute for Quantitative Biomedicine
Rutgers, The State University of New Jersey

Address: 174 Frelinghuysen Road, Piscataway, NJ 08854-8076, United States
Website: https://cv.njzjz.win/
